.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+    "editor.autoFixOnSave": true,
```

Comment:
Maybe we can add **vertical ruler** rule so it will visually show us when we went too far to the right (and our code became unreadable).